doc/src/overrides.md

```diff
@@ -91,9 +91,14 @@ profile = "minimal"
 The `[toolchain]` section is mandatory, and at least one property must be
 specified. `channel` and `path` are mutually exclusive.
 
-For backwards compatibility, `rust-toolchain` files also support a legacy
+For backwards compatibility, the `rust-toolchain` file also supports a legacy
 format that only contains a toolchain name without any TOML encoding, e.g.
-just `nightly-2021-01-21`. The file has to be encoded in US-ASCII this case
+
+```
+nightly-2021-01-21
+```
+
+The file has to be encoded in US-ASCII this case
```

Comment:
I could go either way here, I'm not sure what the block format gains us. If we are switching to that, lets either put all the information about it before the block, rather than splitting it around the block. It is so simple I would be inclined to keep it inline, unless we have evidence of it being systematically confusing.

Reply:
Got it. Let me close it.